Security/Authorization/Interception/MethodSecurityInterceptor.php

```diff
@@ -63,11 +92,19 @@ public function __construct(SecurityContextInterface $securityContext, Authentic
         $this->logger = $logger;
     }
 
+    /**
+     * @param $boolean
+     */
     public function setAlwaysAuthenticate($boolean)
     {
-        $this->alwaysAuthenticate = !!$boolean;
+        $this->alwaysAuthenticate = ! ! $boolean;
     }
 
+    /**
+     * @param MethodInvocation $method
+     * @return mixed
+     * @throws \Exception
```

Comment:
``@throws \Exception`` does not help much here, as it does not describe anything